bemuse/src/auto-synchro/index.tsx

```diff
@@ -0,0 +1,85 @@
+import * as Music from './music'
+
+import ExperimentScene, { ExperimentState } from './ui/ExperimentScene'
+
+import React from 'react'
+import { Subject } from 'rxjs'
+import { createRoot } from 'react-dom/client'
+
+export async function main(bootContext: {
+  setStatus: (status: string) => void
+}) {
+  bootContext.setStatus('Loading song')
+  const state口 = new Subject<ExperimentState>()
+
+  const scene = (
+    <ExperimentScene
+      stateStream={state口.asObservable()}
+      onStart={() => {
+        play()
+        postMessage({ type: 'calibration-started' })
+        addEventListener('beforeunload', () => {
+          postMessage({ type: 'calibration-closed' })
+        })
+      }}
+    />
+  )
+
+  const container = document.createElement('div')
+  document.body.appendChild(container)
+  createRoot(container).render(scene)
+
+  function postMessage(data: object) {
+    if (window.opener) {
+      window.opener.postMessage(data, 'https://bemuse.ninja')
```

Comment:
Recommend `*` because it allows the calibration to be reused in other apps.

```suggestion
      window.opener.postMessage(data, '*')
```